### PR TITLE
Convert jaws/mig64 null values to EMPTY_MENU_ITEM_CHAR

### LINKED ISSUE #227: jaws & mig64 are set incorrectly when duplicating a Rule
I noticed today that when I used "Duplicate" to create a new Rule that jaws & mig64 ended up being set to false, even though the Edit UI showed them as "-". This doesn't happen with the plain "Add" button nor when editing a Rule.

### PR DESCRIPTION
**Pull request by mozbhearsum**
This feels like a horrible hack, but it should fix #227. I'm totally open to a better solution :).

### CODE REVIEW

**Comment by mozbhearsum** [comment]
The idea being that we'd pull the rule info from the backend instead of providing it as state? And I assume we'd use the same actual view code (`views/Rule/index.jsx`) and just add a new route?

**Comment by helfi92** [comment]
Correct!

File: src/components/RuleCard/index.jsx [hunk 1]
```diff
@@ -784,9 +784,8 @@ function RuleCard({
             <Link
               className={classes.link}
               to={{
-                pathname: '/rules/create',
+                pathname: `/rules/${rule.rule_id}/duplicate`,
                 state: {
-                  rule,
                   rulesFilter,
                 },
```
Comment (helfi92) [inline src/components/RuleCard/index.jsx]: Screen Shot 2019-10-10 at 2 11 15 PM

Hmm, clicking "Duplicate" for ^ takes the user to http://localhost:9000/rules/undefined/duplicate. Is there a workaround that comes to your mind to fix this issue?

Comment (mozbhearsum) [reply src/components/RuleCard/index.jsx]: Hm, I didn't realize we had the duplicate button for scheduled additions! The most obvious way to fix this to me is to have two new routes: /rules/duplicate/ruleId/:ruleId and /rules/duplicate/scId/:scId (or something like that). I'll see if I can come up with a better way, though.

Comment (mozbhearsum) [reply src/components/RuleCard/index.jsx]: This turns out to be a pretty good fix, I think, since we already handle ruleId/scId in the params.

**Pull request merged**